src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -66,11 +66,28 @@ private UserStore(PersistentStorageAgent persistentStorageAgent) {
     users = new ArrayList<>();
 
     // hard-coded initial Admin:
-    // TODO(JW): Merge "the creation process" from RegisterServlet to here.
-    String hashedPassword = BCrypt.hashpw("AdminPass01", BCrypt.gensalt());
-    User initialAdmin = new User(UUID.randomUUID(), "Admin01", hashedPassword, Instant.now());
-    initialAdmin.setAdmin(true);
-    users.add(initialAdmin);
+    this.addUser("Admin01", "AdminPass01", /*adminStatus=*/ true);
```

Comment:
Since you changed the parameter name, update the comment to be `/*admin=*/`

Reply:
You are right, I overlooked that. Fixed!